styles.css

```diff
@@ -0,0 +1,174 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+    margin: 0 auto;
+    text-align: center;
+    background-color: #444444;
+}
+
+nav {
+    padding: 20px;
+    background-color: #444444;
+    display: inline-block;
+    width: 100%;
+    height: 100px;
+    position: sticky;
+    top: 0;
+    z-index: 2;
+}
+
+a.anchor {
+    display: block;
+    position: relative;
+    visibility: hidden;
+}
+
+.navLinks a {
+    padding: 20px;
+    margin-right: 50px;
+    text-decoration: none;
+    border: 2px solid #00ffff;
+    color: #00ffff;
+}
+
+.hamburgerMenu div {
+    width: 35px;
+    height: 5px;
+    background-color: #00ffff;
+    border-radius: 12px;
+    margin: 6px auto;
+    display: none;
+}
+
+h1 {
+    font-size: 4vw;
+}
+
+sub {
+    font-size: 2vw;
+}
+
+.contactIcons {
+    margin-top: 40px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+
+}
+
+.gitHub, .twitter, .email, .linkedIn {
+    background-color: #999999;
+    position: relative;
+    width: 10%;
+    padding-top:5.6%;
+}
+
+.homePage, .aboutMe{
+    background-color: #999999;
+    position: relative;
+    width: 100%;
+    padding-top: 56.25%;
+}
+
+.homeText, .aboutText {
+    text-align: center;
+    position: absolute;
+    top: 35%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.portfolioText {
+    background-color: #999999;
+    padding: 25px;
+}
+
+.aboutMeBoxes {
+    width: 100%;
+    height: 100px;
+    position: absolute;
+    top: 75%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+button {
+    width: 300px;
+    height: 100px;
```

Comment:
Height is uneseccary if you are using the transform:translate technique for centering.